### ISSUE TITLE
Gateway not working

### ISSUE THREAD

**Issue by MatthK** [ruuvi/com.ruuvi.station#374]
I have an older Android Phone (Android 5.1.1) that was working just fine until recently.

When I try to upload sensor data through the gateway, it just says "Nope, did not work. Is the URL correct?" when I test the gateway.

I do have a newer Android phone (Android 10) that uses the exact same URL and it works just fine. I suspected, that it might have to do with TLS but even when I lowered the TLS settings on the webserver to accept TLS 1.1 it doesn't work.

I also have an iPhone that I could use for that, but on the iOS app there is no gateway, so that is not a solution either.

Why does the older Android not work anymore? Is there anything I can do to troubleshoot the issue?

Ruuvi Station Version 1.5.16 (on both phones)

**Comment by andreevdenis** [comment]
Do you remember version of app when this feature was working with Android 5.1.1?

I've tried this Android 6.0 phone and it's working as expected. I have Android 5.0 phone but it's barelly working, very slow for some reason, so I wasn't able to test app on it. 

You can try this build on your phone to diagnose: 
https://github.com/ruuvi/com.ruuvi.station/actions/runs/842840107
Install apk from withFilelogsTestApk artifact. Run app and try to test gateway again. App should create log file in Download folder (filename will be {current_date}.log

Search for line "Gateway test exception" in log

**Comment by andreevdenis** [comment]
After hard reset my old device came to life and I was able to reproduce. No need to try test apk for you. I've noticed that test gateway button lead to timeout exception. But usual packets with sensor measurements work as before. Was testing with hookbin.com. Please check if your backend receives data from old phone, just ignore test error.

**Comment by MatthK** [comment]
Strange enough. 

I took the phone home and tried it there. I have turned the phone off while going home, and at home it basically worked again immediately out of the box.

I don't remember 100%, but I thought that I have tried to turn off/turn on again the phone in the office too, so not sure if it that what made the difference. Will try again on Monday at the office to see if there's a problem there.

Thanks for your help.
